pkg/asset/installconfig/azure/validation.go

```diff
@@ -136,6 +136,22 @@ func validateMininumRequirements(fieldPath *field.Path, req resourceRequirements
 	return allErrs
 }
 
+func validateFamily(fieldPath *field.Path, instanceType string, capabilities map[string]string) field.ErrorList {
+	confidentialVMFamilies := sets.NewString(
+		"standardDCASv5Family",
+		"standardDCADSv5Family",
+		"standardECASv5Family",
+		"standardECADSv5Family",
+	)
+	allErrs := field.ErrorList{}
+	if confidentialVMFamilies.Has(capabilities["Family"]) {
+		errMsg := fmt.Sprintf("%s is not currently stupported but will be in a future release", capabilities["Family"])
```

Comment:
Nit: Typo in error message. Should be "supported".

Reply:
Fixed.